packages/desktop/views/login/components/ProfileCardActionsMenu.svelte

```diff
@@ -18,6 +19,23 @@
             false
         )
     }
+
+    function onDeleteClick(): void {
+        openPopup(
+            {
+                id: PopupId.Confirmation,
+                props: {
+                    variant: 'danger',
+                    title: localize('popups.deleteProfile.title'),
```

Comment:
lets add the name of the profile in the title. The ProfileCard in the background isnt highlighted anymore (or can even be under the popup), so it would be good to know what profile this popup is for